script/Ciel/Day 12/12_01_CIEL12_5.txt

```diff
@@ -711,22 +757,22 @@ I'm scared.
 [sha:21e9e2a00171bd2ec0c5b181e99aa37651af6805]{
 -- Page 254, Offset 31634.
 -- 早く鍵をかけないと部屋に入ってくる。
-If I don't lock the door quickly, it will come in.
+If I don't lock the door soon, it'll get in.
 }
 [sha:1a8ba620a90d12e7a62397ed56af8d37be5f6b99]{
 -- Page 254, Offset 31635.
 -- 得体のしれない何かが入ってくる。
-That unknown thing will come inside.
+That unknown thing will get inside.
 }
```

Comment:
doublet! 
入ってくる might need a bit of a more interesting verb.

>If I don't lock the door soon, it will creep in.

>That indistinct %{i}something%{/i} will creep in.

Reply:
Creep feels a bit out of pace with the scene imo. It's a lot slower while he's pretty frantic.

Reply:
>If I don't lock the door soon, it will get in.

>That indistinct %{i}something%{/i} will get in.

Then we may very well go with get, I suppose.